src/functions/types/getMessage.ts

```diff
@@ -12,11 +12,9 @@ export interface messageArg {
   };
 }
 
-export interface sectionArg {
-  text:
-    | string
-    | {
-        type: string;
-        text: string;
-      }[];
-}
+export type sectionArgType =
```

Comment:
```suggestion
export type SectionArgType =
```
type, interface はアッパーキャメルにしたいです

Reply:
self fix